smoke-tests/apps/Cassandra/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/CassandraTest.java

```diff
@@ -45,7 +45,7 @@ void cassandra() throws Exception {
     assertThat(telemetry.rdd1.getName()).isEqualTo("SELECT test.test");
     assertThat(telemetry.rdd1.getData()).isEqualTo("select * from test.test");
     assertThat(telemetry.rdd1.getType()).isEqualTo("cassandra");
-    assertThat(telemetry.rdd1.getTarget()).matches("dependency[0-9]+");
+    assertThat(telemetry.rdd1.getTarget()).matches("[0-9.]+");
```

Comment:
this feels like a regression, I think we need the upstream lettuce after all, can you update the upstream PR to implement both Network and Server attributes getter?

Reply:
server is green.. i can add network.